.github/workflows/simulation-1.2.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       CXXFLAGS: -${{ matrix.arch }}
       LDFLAGS: -${{ matrix.arch }}
       COVERAGE: 1
-      THREAD_VERSION: 1.2
+      THREAD_VERSION: inter-op
```

Comment:
Better not change the original behavior? 
Since the valid Thread versions are 1.1 and 1.2, we can use another variable to specify the intention to only build 1.2?

Reply:
I think it's more intuitive that if `THREAD_VERSION` is `1.1` then we only build 1.1 and if it's `1.2` then we only build `1.2`. I think it's a little confusing if `1.1` builds only `1.1`, `1.2` builds them all, and another variable makes it build only `1.2`. Thoughts? There won't be big changes on our test configurations.

Reply:
I think it would be better to add a VARIABLE `INTER-OP`. If it's `1` and `THREAD_VERSION` is `1.2` we build both versions. Otherwise, it only builds `1.2`.